### PR TITLE
Fix rest of the problems of building on arm

### PR DESCRIPTION
**Pull request by rickif**
This PR is to solve the rest problems of buiding on aarch64, including the following modifications.
  1. new crc_hash functions adaption
  2. hadoop for aarch64
  3. OpenJDK for aarch64
  4. correcting Apache Arrow 5.0.0 building option
  5. use CDN to improve the download speed of third-party libraries
  6. fix oracle release repository. ref #153

### CODE REVIEW

**Review by dirtysalt — DISMISSED** [review]

**Comment by dirtysalt** [comment]
Thanks for this huge work of porting code to arm.

File: thirdparty/vars.sh [hunk 1]
```diff
@@ -253,12 +253,24 @@ HADOOP_NAME="hadoop-3.3.0.tar.gz"
 HADOOP_SOURCE="hadoop-3.3.0"
 HADOOP_MD5SUM="c4ba5155ec44abcb63d183d2dbe38df9"
 
+# HADOOP_AARCH64
+HADOOP_AARCH64_DOWNLOAD="http://starrocks-thirdparty.oss-cn-zhangjiakou.aliyuncs.com/hadoop-3.3.0-aarch64.tar.gz"
```
Comment (Astralidea) [inline thirdparty/vars.sh:257]: Is it better to change this to CDN?

**Review by imay — DISMISSED** [review]

**Review by chaoyli — APPROVED** [review]

**Review by chaoyli — DISMISSED** [review]

File: thirdparty/vars-aarch64.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```
Comment (chaoyli) [inline thirdparty/vars-aarch64.sh:1]: use elastic license

File: thirdparty/vars-x86_64.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```
Comment (chaoyli) [inline thirdparty/vars-x86_64.sh:1]: use elastic license

**Review by chaoyli — CHANGES_REQUESTED** [review]

File: thirdparty/vars-aarch64.sh (hunk 2 above)
Comment (rickif) [reply thirdparty/vars-aarch64.sh:1]: Already changed to elastic license 2.0

File: thirdparty/vars-x86_64.sh (hunk 3 above)
Comment (rickif) [reply thirdparty/vars-x86_64.sh:1]: Already changed to elastic license 2.0

**Review by imay — APPROVED** [review]

**Review by chaoyli — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Pull request merged**